README.md

```diff
@@ -4,8 +4,11 @@ Code Climate Engine to run [markdownlint][mdl]
 
 ## Installation
 
+[![Code Climate](https://codeclimate.com/github/codeclimate/codeclimate-markdownlint/badges/gpa.svg)](https://codeclimate.com/github/codeclimate/codeclimate-markdownlint)
```

Comment:
Might want to move these out of the Installation section and right below the description on line 3?